variants/FYSETC_S6/PinNamesVar.h

```diff
@@ -0,0 +1,30 @@
+/* SYS_WKUP */
+#ifdef PWR_WAKEUP_PIN1
+SYS_WKUP1 = PA_0, /* SYS_WKUP0 */
+#endif
+#ifdef PWR_WAKEUP_PIN2
+SYS_WKUP2 = NC,
```

Comment:
Could you update this file with the last generated one:
https://github.com/stm32duino/Arduino_Tools/tree/master/src/genpinmap/Arduino/STM32F4/STM32F446V(C-E)Tx

For example it misses the SYS_WKUP2 and some definitions

Reply:
You mean we need other USB pins def ?

Reply:
This is only definition. I would avoid further support because it was not defined even if currently you don't use it.
At least you can just add and comment them like this user will be warned which one exists.

Reply:
Updated.